corehq/ex-submodules/pillowtop/migrations/0004_rename_synclog_checkpoint.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db.migrations.operations.special import RunSQL
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pillowtop', '0003_auto_20170411_1957'),
+    ]
+
+    operations = [
+        # in case there's a rollback we want to override any existing value
+        RunSQL("DELETE FROM pillowtop_djangopillowcheckpoint WHERE checkpoint_id = 'synclogs'"),
```

Comment:
I think you'll want synclogs_2017-11-01 as the checkpoint id.

I think there are ways to use the models instead of raw sql. any reason you chose to use raw sql here?

Reply:
I was already in SQL mode from writing the query below. No other reason.

This migration isn't strictly necessary since the new checkpoint will get created automatically, this is more or a backup but not that I think about it more I don't think it's that helpful in any situation.